pytorch_blade/benchmark/TorchBench/test_torch_bench.sh

```diff
@@ -31,23 +31,32 @@ pushd $script_dir # pytorch_blade/benchmark/TorchBench
 ln -s $benchmark_repo_dir torchbenchmark
 
 # benchmark 
-config_file=blade_bench.yaml
-bench_target=partial
-if [ $1 ]; then
-    config_file=blade_bench_$1.yaml
-    bench_target=$1
-fi
+config_file=blade_$1_$2.yaml
+bench_target=$2
 # setup benchmark env
-export DISC_ENABLE_STITCH=true DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE=true \
+export DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE=true \
     DISC_CPU_LARGE_CONCAT_NUM_OPERANDS=4 DISC_CPU_ENABLE_EAGER_TRANSPOSE_FUSION=1 \
-    OMP_NUM_THREADS=1 TORCHBENCH_ATOL=1e-2 TORCHBENCH_RTOL=1e-2 TORCH_DISC_USE_TORCH_MLIR=true
+    OMP_NUM_THREADS=1 TORCHBENCH_ATOL=1e-2 TORCHBENCH_RTOL=1e-2
+if [ $2 == "cpu" ];then
+    # 4 cores
+    export GOMP_CPU_AFFINITY="1-4"
```

Comment:
core 1 to core 4 may distribute across physical core 0, 1, 2 ... why not bind to 2-5, fully leveraging physical core 1 and 2?

Reply:
OK~